lib/web/integrations_awsra.go

```diff
@@ -0,0 +1,136 @@
+func (h *Handler) awsRolesAnywhereConfigureTrustAnchor(w http.ResponseWriter, r *http.Request, p httprouter.Params) (any, error) {
+	ctx := r.Context()
+
+	queryParams := r.URL.Query()
+
+	integrationName := queryParams.Get("integrationName")
+	if integrationName == "" {
+		return nil, trace.BadParameter("missing integrationName param")
+	}
+
+	trustAnchorName := queryParams.Get("trustAnchor")
+	if trustAnchorName == "" {
+		return nil, trace.BadParameter("missing trustAnchor param")
+	}
+	if err := aws.IsValidIAMRolesAnywhereTrustAnchorName(trustAnchorName); err != nil {
+		return nil, trace.BadParameter("invalid trustAnchor %q", trustAnchorName)
+	}
+
+	syncRoleName := queryParams.Get("syncRole")
+	if syncRoleName == "" {
+		return nil, trace.BadParameter("missing syncRole param")
+	}
+	if err := aws.IsValidIAMRoleName(syncRoleName); err != nil {
+		return nil, trace.BadParameter("invalid role %q", syncRoleName)
+	}
+
+	syncProfileName := queryParams.Get("syncProfile")
+	if syncProfileName == "" {
+		return nil, trace.BadParameter("missing syncProfile param")
+	}
+	if err := aws.IsValidIAMRolesAnywhereProfileName(syncProfileName); err != nil {
+		return nil, trace.BadParameter("invalid syncProfile %q", syncProfileName)
+	}
+
+	clusterName, err := h.GetProxyClient().GetDomainName(ctx)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	// Ensure the IntegrationName is valid.
+	_, err = h.GetProxyClient().GetIntegration(ctx, integrationName)
+	// NotFound error is ignored to prevent disclosure of whether the integration exists in a public/no-auth endpoint.
+	if err != nil && !trace.IsNotFound(err) {
+		return nil, trace.Wrap(err)
+	}
+
+	authorities, err := client.ExportAllAuthorities(
+		ctx,
+		h.GetProxyClient(),
+		client.ExportAuthoritiesRequest{
+			AuthType: string(types.AWSRACA),
+		},
+	)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	if len(authorities) == 0 {
+		return nil, trace.NotFound("no AWS IAM Roles Anywhere CA found")
+	}
+
+	awsRACACertB64 := base64.RawStdEncoding.EncodeToString(authorities[0].Data)
+
+	// The script must execute the following command:
+	// teleport integration configure awsra-trust-anchor
+	argsList := []string{
+		"integration", "configure", "awsra-trust-anchor",
+		fmt.Sprintf("--cluster=%s", shsprintf.EscapeDefaultContext(clusterName)),
+		fmt.Sprintf("--name=%s", shsprintf.EscapeDefaultContext(integrationName)),
+		fmt.Sprintf("--trust-anchor=%s", shsprintf.EscapeDefaultContext(trustAnchorName)),
+		fmt.Sprintf("--sync-profile=%s", shsprintf.EscapeDefaultContext(syncProfileName)),
+		fmt.Sprintf("--sync-role=%s", shsprintf.EscapeDefaultContext(syncRoleName)),
+		fmt.Sprintf("--trust-anchor-cert-b64=%s", awsRACACertB64),
+	}
+
+	script, err := oneoff.BuildScript(oneoff.OneOffScriptParams{
+		EntrypointArgs: strings.Join(argsList, " "),
+		SuccessMessage: "Success! You can now go back to the Teleport Web UI to continue the set up.",
```

Comment:
```suggestion
		SuccessMessage: "Success! You can now go back to the Teleport Web UI to continue the setup.",
```
In this case you want the noun.